src/actions/requests.rs

```diff
@@ -477,24 +478,24 @@ impl<'a> RequestAction<'a> for Rename {
 pub struct Deglob;
 
 impl Deglob {
-    fn calculate_workspace_edit(&mut self, location: &<Self as Action>::Params, ctx: &ActionContext) -> Result<DeglobResult, (ErrorCode, &'static str)> {
+    fn calculate_workspace_edit(
+        &self,
```

Comment:
I meant that this doesn't need `&self` at all, same with `make_..._actions()`, but I wouldn't want to block this on that